app/_locales/en/messages.json

```diff
@@ -2648,6 +2648,12 @@
   "usePhishingDetectionDescription": {
     "message": "Display a warning for phishing domains targeting Ethereum users"
   },
+  "useStaticTokenList": {
+    "message": "Token detection"
```

Comment:
Each other setting in this panel starts with an action word, so I think we should change this to "Use Token Detection"